treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -60,7 +60,7 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity):
 
     with transaction.atomic():
         for datum in series:
-            PerformanceDatum.objects.create(
+            PerformanceDatum.objects.get_or_create(
```

Comment:
Actually that doesn't seem to play well with atomic transactions. Forget it, I'm just going back to the original idea. :)